neo/Ledger/BlockChain.State.cs

```diff
@@ -13,7 +13,7 @@ namespace Neo.Ledger
     public sealed partial class Blockchain : UntypedActor
     {
         public class ImportRoots { public IEnumerable<StateRoot> Roots; }
-        public uint StateHeight => currentSnapshot.StateHeight;
+        public long StateHeight => currentSnapshot.StateHeight == uint.MaxValue ? -1 : (long)currentSnapshot.StateHeight;
```

Comment:
we should use long in `currentSnapshot.StateHeight` or check if it's something stored instead of check `uint.MaxValue`